src/dali_executor/dali_pipeline.h

```diff
@@ -160,9 +164,11 @@ class DaliPipeline {
   }
 
   void ReleaseStream() {
+    if (no_gpu_)
```

Comment:
If the output is on the CPU does backend still works on stream?

Reply:
IIUC, you're referring to the output of DALI. It might, if the Triton requested GPU output and DALI provides CPU. Since we get to know Triton's requested output in runtime, we need to have the stream ready just in case